### DISCUSSION TITLE
Left hand and Right hand modes

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#913] [Ideas] [answered]
Hey I'd really like to have a right hand and left hand mode, where the only characters that have to be typed can be reached with that hand. Not sure how it would work, like you could grey out the other letters or color them red or orange or simply not include them. But hiding them wouldn't work well with quotes(or anything readable) so I think coloring them out is better, even though It may be confusing at first.  I would love to have that cursor continuation like a robot is typing the other hand's letters for you.

Another option, is Including "English Right/Left hand" word lists just like English 450k, but I think having right hand and left hand modes plus that is the best option. You could scrap a dictionary for those words composed solely of the characters that are supposed to be clicked with each hand to get the word lists.

The reasoning is that sometimes one hand is strained while the other is well and you'd want to give one hand a rest while working with the other. Also good for people who have broken an arm or a hand. Also for those buttons that are reachable with both hands this can help you be consistent with which finger you use to type. And also I think it will be fun, maybe more than the far galaxy mode.

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
A while ago Miodec showed this [word filter](https://gfruit.github.io/typing/words-filter.html) which does a lot like filtering words that have specific letters. There's also another feature that gives you words that only use certain fingers. 
![Screenshot 2021-02-04 at 8 43 42 AM](https://user-images.githubusercontent.com/58377021/106917078-2200cb80-66c5-11eb-9425-1f232f4a8285.png)
lp stands for "left pinky" and lr stands for "left ring finger" so using this you could enable every finger on the left side then filter out words that use those fingers. It's a pretty neat filter
